Tribler/Test/Core/Modules/test_process_checker.py

```diff
@@ -1,12 +1,14 @@
 import os
-from multiprocessing import Process
+from multiprocessing import Process, Value
+from time import sleep
 
 from Tribler.Core.Modules.process_checker import ProcessChecker, LOCK_FILE_NAME
 from Tribler.Test.test_as_server import AbstractServer
 
 
-def process_dummy_function():
-    while True:
+def process_dummy_function(stop_flag):
+    while stop_flag.value == 0:
+        sleep(0.001)
```

Comment:
Granularity of sleep is only in 10's of ms (0.01).

Reply:
ow!